docs/source/developing/writing-docs.adoc

```diff
@@ -101,4 +172,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you check-in the updated tarball,
```

Comment:
"After you check-in the updated tarball,"? We are not checking in a tarball. I guess you are referring to http://artifacts.ci.centos.org/minishift/minishift/docs/latest/. This tarball is created on request by a CI job.